Sat, 18 Jan 2020 17:14:21 - INFO - Connect to DB
Sat, 18 Jan 2020 17:14:21 - INFO - TRIZMA DB import
Sat, 18 Jan 2020 17:14:21 - INFO - Failed on TRIZMA DB import at: 2020-01-18_17-00-52, for following calling party: 4921115770460, for agent id: trz_asilni, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_asilni',                                 '4921115770460','','Nicht übermittelt','4921115770460_Saturday, January 18, 2020_6:14:02 PM','ANA_SILNI')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Sat, 18 Jan 2020 17:14:21 - INFO - 123TV DB import
Sat, 18 Jan 2020 17:15:52 - INFO - Connect to DB
Sat, 18 Jan 2020 17:15:52 - INFO - TRIZMA DB import
Sat, 18 Jan 2020 17:15:52 - INFO - 123TV DB import
Sat, 18 Jan 2020 17:20:30 - INFO - Connect to DB
Sat, 18 Jan 2020 17:20:30 - INFO - TRIZMA DB import
Sat, 18 Jan 2020 17:20:30 - INFO - 123TV DB import
Sat, 18 Jan 2020 17:21:37 - INFO - Connect to DB
Sat, 18 Jan 2020 17:21:37 - INFO - TRIZMA DB import
Sat, 18 Jan 2020 17:21:37 - INFO - 123TV DB import
